Bender.yml

```diff
@@ -56,6 +56,7 @@ sources:
     - hardware/src/lane/lane.sv
     # Level 5
     - hardware/src/ara.sv
+    - hardware/src/accel_dispatcher_ideal.sv
```

Comment:
I believe this file is not synthesizable, right? Wouldn't it make more sense to have it in another section of the Bender.yml file?

Reply:
I am also looking forward to seeing how the new roofline plots look like. Waiting for the CI.. :popcorn:

Reply:
Me too! But for that, we should wait the support for Verilator :(